src/index.js

```diff
@@ -0,0 +1,20 @@
+// import _ from 'lodash';
+import './style.css';
+
+const input = document.getElementById('input');
+const addBtn = document.getElementById('addBtn');
+const list = document.getElementById('list');
+
```

Comment:
- [ ] The should be an array of tasks with each task represented as an object. The task array would contain some tasks as placeholders for now:
Eg:
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];

```